### ISSUE TITLE
Update latest release section for 0.39.0

### ISSUE THREAD

**Issue by Sreekala-Gopakumar** [eclipse-openj9/openj9-website#352]
What's New update for the website, for 0.39.0 (June 2023)

**Comment by Sreekala-Gopakumar** [comment]
First draft (0.38.0 content is live at https://www.eclipse.org/openj9/news). Note: this isn't an exhaustive list of all the changes, just the main ones:

> ### Eclipse OpenJ9 version 0.39.0 released
> June 2023
> 
> We're pleased to announce the availability of Eclipse OpenJ9&trade; v0.39.0.
> 
> This release supports OpenJDK 20. For more information about supported platforms and OpenJDK versions, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
>
> Other updates in this release include the following:
> 
> - 
>
> To read more about these and other changes, see the [OpenJ9 user documentation](https://www.eclipse.org/openj9/docs/openj9_releases/).
> 
> #### Performance highlights include:
> 
> - 
>

**Comment by Sreekala-Gopakumar** [comment]
@pshipton @vijaysun-omr @tajila @babsingh Are there any performance updates that you want to highlight in the website?

**Comment by babsingh** [comment]
> Are there any performance updates that you want to highlight in the website?

Virtual threads have been significantly optimized for better throughput and memory utilization.

@Sreekala-Gopakumar Is the above statement sufficient or are more details required?

**Comment by vijaysun-omr** [comment]
@amicic was footprint improvement due to dual indexable header shape shipped in this release ? If so, we should mention a sentence about the fact that footprint improved.

**Comment by vijaysun-omr** [comment]
We should mention that "MethodHandle throughput performance was improved by some changes in this release"

**Comment by vijaysun-omr** [comment]
The change in question was https://github.com/eclipse-openj9/openj9/pull/17322 . Instead of "some changes" you could say "using compiled code instead of interpreter in more cases" and maybe that is the right balance between being too specific and too generic.

**Comment by amicic** [comment]
> @amicic was footprint improvement due to dual indexable header shape shipped in this release ? If so, we should mention a sentence about the fact that footprint improved.

Yes, I can confirm that dual indexable header was enabled in 0.39.

<img width="663" alt="image" src="https://github.com/eclipse-openj9/openj9-website/assets/18264984/8a725af1-620d-4d1f-a4e3-46842edd892c">

**Comment by amicic** [comment]
Besides small footprint improvements (or reduction of GC overhead for fully expanded heap) in GC policies other than Balanced GC, note there is also a slight throughput regression in Balanced GC, since we now initialize dataAddr. But we intend to compensate it with off-heap allocations in near future.

**Comment by babsingh** [comment]
LGTM. Minor reword recommended:
> For virtual threads, throughput is optimized by reducing the synchronization overhead in the mount and unmount code paths, and memory usage is reduced through a Java stack caching feature that enables reuse of Java stacks.

**Comment by vijaysun-omr** [comment]
Thanks, LGTM

**Comment by Sreekala-Gopakumar** [comment]
@pshipton @vijaysun-omr @tajila @babsingh @amicic - Can we finalise this content? Thanks!

**Comment by Sreekala-Gopakumar** [comment]
> LGTM. Minor reword recommended:
> 
> > For virtual threads, throughput is optimized by reducing the synchronization overhead in the mount and unmount code paths, and memory usage is reduced through a Java stack caching feature that enables reuse of Java stacks.

Done.

**Comment by babsingh** [comment]
> Can we finalise this content?

GTG. No more changes from my end.
